Source/Core/Core/Core.vcxproj.filters

```diff
@@ -175,6 +175,7 @@
     <ClCompile Include="Movie.cpp" />
     <ClCompile Include="NetPlayClient.cpp" />
     <ClCompile Include="NetPlayServer.cpp" />
+    <ClCompile Include="NetPlayQoS.cpp" />
```

Comment:
The header file should be placed with the rest of the header files in this file as well.

Reply:
Fixed.